warrior/Actions/CommonActions/common_actions.py

```diff
@@ -228,4 +237,59 @@ def set_env_var(self, var_key=None, var_value=None, filepath=None,
 
         return status
 
+    def get_values_from_datafile(self, system_name, strvar, langs, states,
+                                 currencys, ramspace, configfile, intvar,
+                                 file_config):
```

Comment:
I see this kw is for testing purpose not for generic use, name 'get_values_from_datafile' says it is to get values but it actually checks the type and returns the status and also the argument names are not generic & the docstring doesn't have any info of how the datafile should be written(using type) for this kw. What would happen if the user doesn't give 'type' tag in the datafile and use this keyword with wtag.

Reply:
Done

Reply:
Completed.